integrations/dd-sdk-android-sqldelight/api/dd-sdk-android-sqldelight.api

```diff
@@ -1,3 +1,10 @@
+public final class com/datadog/android/sqldelight/BuildConfig {
```

Comment:
where is this class being generated if it was picked up by apiSurface ?

Reply:
This was comming from the snippet in the shared `androidLibraryConfig()` I created. 
```
        buildFeatures {
            buildConfig = true
        }
```

I removed this to cleanup this change